scripts/test_motor.py

```diff
@@ -0,0 +1,46 @@
+from easydriver.easydriver import EasyDriver, PowerState, MicroStepResolution,  StepDirection
+from gpiozero.pins.rpigpio import RPiGPIOFactory
+from gpiozero import Device
+import time
+import RPi.GPIO as rpigpio
+
+if __name__ == "__main__":
+    STEP_PIN = 20
+    DIR_PIN = 21
+    ENABLE_PIN = 25
+    MS1_PIN = 24
+    MS2_PIN = 23
+
+    rpigpio.setmode(rpigpio.BCM)
+    rpigpio.setwarnings(False)
+
+    Device.pin_factory = RPiGPIOFactory()
+
+    driver = EasyDriver(step_pin=STEP_PIN,
+                dir_pin=DIR_PIN, 
+                ms1_pin=MS1_PIN, 
+                ms2_pin=MS2_PIN,
+                enable_pin=ENABLE_PIN)
+
+    print("full-step fwd")
+    driver.microstep_resolution = MicroStepResolution.FULL_STEP
```

Comment:
How do the resolutions work? When we do like half step, do we actually have 400/steps per rotation?

Reply:
Basically, microstepping can half the step of your angle each step. full step = 1.8 degrees. half-step = 0.9 degress. quarter-step = 0.45 degrees.

I don't find that half-step and quarter-step work all that well though. It seems that full-step works more reliably.